src/main/java/org/gridsuite/modification/server/repositories/NetworkModificationRepository.java

```diff
@@ -51,8 +52,7 @@ public void saveModifications(UUID groupUuid, List<? extends ModificationEntity>
     }
 
     @Transactional
-    // To have all move in the same transaction (atomic)
-    // When we move modifications, we move them right before referenceModification when it is defined, at the end of list otherwise
+    // TODO Remove transaction when errors will no longer be sent to the front
     public List<ModificationEntity> moveModifications(UUID destinationGroupUuid, UUID originGroupUuid, List<UUID> modificationsUuid, UUID referenceModificationUuid) {
```

Comment:
the caller (NetworkModificationRepository::moveModifications) is now Transactional, so why not removing Transactional here now ?